guidance/config/params.yaml

```diff
@@ -20,3 +20,5 @@ spin_rate_hz: 10
 required_plugins:
   - Autoware Plugin 1.0.0
   - Pure Pursuit Plugin 1.0.0
+
+vehicle_type: cadilac # lexus or cadilac
```

Comment:
This parameter is an aspect of the vehicle configuration and should therefore be stored in the CARMAConfig repo. I would suggest renaming it to something like vehicle_state_machine_type or something similar and adding it to the 
VehicleConfigParams.yaml file located in the appropriate sub directories of that repo.

Reply:
I do not know where is the yaml file you are mentioning is located can you send the link to that file ?

Reply:
There is one in nearly every directory of the repo just under the top level folder. You need to add it to all of them

Reply:
we need to discuss this further.